build.gradle

```diff
@@ -1,13 +1,29 @@
 plugins {
-    id 'eu.xenit.docker' version '5.1.0' apply false
-    id 'eu.xenit.docker-compose' version '5.1.0' apply false
+    id 'eu.xenit.docker' version '5.1.1' apply false
+    id 'eu.xenit.docker-compose' version '5.1.1' apply false
 }
 
 repositories {
     mavenCentral()
     jcenter()
 }
 
+String getBranchName(Project project) {
+    return project.hasProperty('branchName') ? project.branchName : (System.getenv('BRANCH_NAME') ?: 'local')
+}
+
+String getTagQualifier(String branchName) {
+    if (branchName.endsWith('release')) {
```

Comment:
Shouldn't we check for `refs/heads/release` here?
Else, we are also going to match a branch named `blabla-release` or `prepare-next-release`.

But if we are going to do that, we run into a problem with the scheduled build, where `BRANCH_NAME` is set to just `release` or `master`, so the value of `BRANCH_NAME` has to be fixed there as well.

Reply:
How about `if (branchName == 'release' || branchName == 'refs/heads/release')`?

Reply:
I was thinking more like updating `scheduled-build.yml` to have it's matrix be `refs/heads/release` and `refs/heads/master` instead of `release` and `master`

Reply:
I tried this but that does not work with ```
- uses: actions/checkout@v2
        with:
          ref: ${{ matrix.branch }}
```
I agree using same input in workflows files is a good idea, so I'll change it before the gradle call.